_playwright-tests/UI/helpers/repoHelpers.ts

```diff
@@ -0,0 +1,8 @@
+export const randomName = () => (Math.random() + 1).toString(36).substring(2, 6);
+
+export const randomNum = () =>
+  Math.floor(Math.random() * 100 + 1)
+    .toString()
+    .padStart(2, '0');
+
+export const randomUrl = () => `https://stephenw.fedorapeople.org/centirepos/repo${randomNum()}/`;
```

Comment:
this is nice, much simpler this way 👍🏼 

but it would be even better to have those repos in our [fixtures repo](https://github.com/content-services/fixtures/), which was created for this exact purpose, i.e.: hosting our's testing repositories, where the ownership would be on the entire team 😄

Reply:
RFE versus bugfix

Reply:
Hi, I think this might work:

https://github.com/content-services/fixtures/pull/6

Reply:
that would, awesome! 👏🏼 
just the URL needs to be changed now to point to the new location 😄

Reply:
@dominikvagner rebased and updated to use github repo path. 
Thank you.

Reply:
Changes to repo path also applied in:

https://github.com/content-services/content-sources-frontend/pull/495

Reply:
If you merge https://github.com/content-services/content-sources-frontend/pull/495 we can close this PR.